src/commands/new.js

```diff
@@ -70,6 +71,12 @@ const questions = [
       return val.toLowerCase()
     },
     default: 'optional'
+  },
+  {
+    type: 'confirm',
+    name: 'git',
+    message: 'Do you want to initialize a Git repository?',
+    default: true
```

Comment:
I think the default should be false here.

Reply:
Ok

Reply:
Done